src/login.c

```diff
@@ -402,6 +437,13 @@ int login_handler(struct worker_t *self, struct client_t *c, int l4proto, char *
 		}
 	}
 	
+
+	if (disallow_unverified && !(c->validated)) {
```

Comment:
This is not right; this breaks connections from unverified clients. They are intentionally allowed to connect, they're just not allowed to send packets. There might be some yelling if this change is done as a surprise; if you wish to propose a change like this, please do so on APRSSIG.